test/suite.bats

```diff
@@ -22,14 +22,14 @@
 }
 
 
-@test "ruby version is 2.3(debian) or 2.5(alpine)" {
+@test "ruby version is 2.6(debian) or 2.5(alpine)" {
   run docker run --rm $IMAGE sh -c "ruby --version | cut -d ' ' -f 2"
   [ "$status" -eq 0 ]
 
   major=$(echo "$output" | cut -d '.' -f 1)
   minor=$(echo "$output" | cut -d '.' -f 2)
   [ "$major" -eq "2" ]
-  [[ "$minor" =~ [3-5]+ ]]
+  [[ "$minor" =~ [3-6]+ ]]
```

Comment:
I think `[5-6]` is better since the description of the test is `ruby version is 2.**6**(debian) or 2.**5**(alpine)` .